### DISCUSSION TITLE
why you don't use rust ??

### DISCUSSION THREAD

**Discussion by ghost** [oven-sh/bun#7853] [Q&A]
Hello Bun Team ...
why you don't use rust and use zig ??? please explain it deeply ... 
 we want to learn from your experience

**Comment by ghost** [comment]
@Jarred-Sumner please answer

**Reply by shawshankkumar** [reply]
He has answered it on his Twitter account.

**Comment by zaxx-q** [comment]
your wish is his command
https://github.com/oven-sh/bun/pull/30412
